exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -163,7 +177,12 @@ func newMetricsExporter(config Config, set component.ExporterCreateSettings, mar
 		return nil, err
 	}
 
+	if err := view.Register(MetricViews()...); err != nil {
```

Comment:
It looks like the same views can be registered multiple times if multiple signal exporters are configured.  Does that work correctly?

Reply:
The code [here](https://github.com/census-instrumentation/opencensus-go/blob/v0.23.0/stats/view/worker.go#L318-L324) seems to be able to take care of this situation. If they're the same it will be considered success and won't re-register.

Also manually tested and it seems ok. (setup 2 pipeline of trace and metrics, and the resulting metrics are correct).